metabulo/views.py

```diff
@@ -180,10 +180,11 @@ def set_normalization_method(csv_id):
     csv_file = CSVFile.query.get_or_404(csv_id)
     args = request.json
     method = args['method']
-    if method is not None and method not in NORMALIZATION_METHODS:
-        raise ValidationError('Invalid normalization method', data=method)
+    argument = args.get('argument', None)
     try:
+        validate_normalization_method(method, argument)
```

Comment:
It would be better to do this with webargs.

Reply:
Sure, I can move this call into the validator.

Reply:
Nevermind, I didn't realize the use_kwargs decorator took all the same kwargs that parser.parse does.  There are no examples explaining that in the docs.